data/data/gcp/cluster/dns/base.tf

```diff
@@ -9,6 +9,7 @@ resource "google_dns_managed_zone" "int" {
   description = local.description
   dns_name    = "${var.cluster_domain}."
   visibility  = "private"
+  project     = var.project_id
```

Comment:
Why do we need to add project_id for these resources (when we previously didn't need them)? Was the project being inferred in some way?